modules/8_reinforcement_learning/README.md

```diff
@@ -0,0 +1,26 @@
+# Reinforcement Learning
+
+This module asks to solve one of `gym` environments.
+Check out `try_gym()` function to get perception of what `gym` is about.
+To complete this module you should:
+1. implement
+    i. `Agent`'s methods. Hyperparamthers `ALPHA`, `GAMMA` are initialized for you
+    ii. A part of `train()` function
+2. Add `q_values.npy` file saved by `agent.save(WEIGHTS_PATH)`
+
+## Details
+
+* Install dependencies:
+
+```sh
+python3 -m pip install -r requirements.txt
+```
+
+* Modify [`reinforcement_learning.py`](reinforcement_learning.py) and run it. It takes about 10 minutes to train.
+* Run a test Python script:
+
+```sh
+python3 test_reinforcement_learning.py
+```
+
+* Add `q_values.npy` file
```

Comment:
* Add resulting `q_values.npy` file to commit

Reply:
Updated